<ui version="4.0" >
 <class>StaffTextProperties</class>
 <widget class="QDialog" name="StaffTextProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MuseScore: Staff Text Properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="channel" >
     <property name="title" >
      <string>Channel</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QListWidget" name="channelList" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="midiAction" >
     <property name="title" >
      <string>Midi Action</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <widget class="QListWidget" name="midiActionList" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StaffTextProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StaffTextProperties</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
